Misc/NEWS.d/next/Library/2020-02-23-15-09-47.bpo-39244.aBK5IM.rst

```diff
@@ -0,0 +1,2 @@
+Fixed :class:`multiprocessing.context.get_all_start_methods`
+If the context is macOS, return the default methods by order.

```

Comment:
This wording is imprecise; this should always return the default method first, which was not done on the macOS platform.

Reply:
Done.